parameter/log/include/log/LogWrapper.h

```diff
@@ -83,33 +83,38 @@ class LogWrapper
     /**
      * Simulate stream behaviour
      *
+     * Both operator<<() versions must not be used in conjunction: either use
+     * the 'scalar' version (this one) or the 'list' version but not both on
+     * the same object.
+     *
      * @tparam T the type of the information to log
      * @param[in] log the information to log
      */
     template <class T>
     LogWrapper& operator<<(const T &log)
     {
-        mLog << log;
+        if (mLogs.empty()) {
+            mLogs.emplace_back();
+        }
+
+        mLogs.back() << log;
```

Comment:
maybe we could add std::to_string instead of an ostringstream ?
... Never mind, `std::to_string(std::string)` <=> `id` does not exist. Unfortunately.